src/parse/deframe/bytes_until_eof.rs

```diff
@@ -0,0 +1,27 @@
+use std::io::BufRead;
+
+use crate::parse::deframe::deframe::{Deframe, DeframerResult};
+
+pub struct BytesUntilEofDeframer {
```

Comment:
Does this need to be a separate file from bytes?

Reply:
Good point. Yeah I think it makes more sense to combine it with bytes too.

Reply:
Cool. I think it makes more sense organizationally speaking